osu.Game/Overlays/UserProfileOverlay.cs

```diff
@@ -179,16 +216,73 @@ private void userLoadComplete(User user)
                     sec.User.Value = user;
 
                     sectionsContainer.Add(sec);
-                    tabs.AddItem(sec);
+                    sectionTabs.AddItem(sec);
                 }
             }
+
+            string playMode;
+            switch (user.PlayMode)
+            {
+                default:
+                case "osu":
+                    playMode = "osu!";

```

Comment:
these should be retrieved from RulesetStore. we don't want to have string constants across the code base